<launch>

<node name = "base_controller" pkg="tom_3d_control" type="base_controller" output="screen">

	</node>
</launch>
